exercises/reverse-string/description.md

```diff
@@ -0,0 +1,16 @@
+# Reverse String
+
+The introductory exercise to reverse an input string.
+
+This is a typical program for beginning programming in a new language
+or environment after being able to do simple tasks.
+
+The objectives are simple:
+
+- Write a function that reverses the input string.
+- Test various mixed case inputs and output the reverse of them.
+- Run the test suite and make sure that it succeeds.
+- Submit your solution and check it at the website.
+
+Comments: 
+If testing tools are available, a property that can be tested is to reverse string twice and see if the output is the original string.
```

Comment:
should not be in this file. it is in the JSON file, the right place, and students will only be confused by this "If", because the question will be "Aren't you providing the tests for me? Why are you telling me it *can* be tested?"

Reply:
I made updates, it's only in JSON file, please let me know of any edit suggestions. Thanks.